AnimalKingdom/src/animalKingdom/Animal.java

```diff
@@ -0,0 +1,36 @@
+package animalKingdom;
+
+public abstract class Animal {
+
+    String name;
+    int Id;
+    int year;
+
+    public Animal(String name, int Id, int year) {
+        this.name = name;
+        this.Id = Id;
+        this.year = year;
+    }
+
+    // Abstract methods
+    public abstract String getName();
+
+    public abstract String move();
+
+    public abstract String breath();
+
+    public abstract int getYear();
+
+    public abstract String reproduce();
+
+    // Regular Methods
+    public void eat(String food) {
+        System.out.println("The " + this.name + "eats " + food);
+    }
```

Comment:
Nice comment -  shows good distinction between what operations are supposed to have abstract methods versus methods that should be common to all instances (including subclasses).